presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -125,6 +126,8 @@ public ExchangeClient(
             Executor pageBufferClientCallbackExecutor)
     {
         this.bufferCapacity = bufferCapacity.toBytes();
+        this.averageBytesPerRequest = this.bufferCapacity / initialRequestCount;
```

Comment:
check state `initialRequestCount > 0` if you want

Reply:
Added a check for averageBytesEMAAlpha (not adding `initalRequestCount` anymore).